json-schema/python/frinx_api/json_schema/device_removal.py

```diff
@@ -0,0 +1,20 @@
+# generated by datamodel-codegen
+
+from __future__ import annotations
+
+from pydantic import BaseModel
+from pydantic import ConfigDict
+
+
+class DeviceRemoval(BaseModel):
+    """
+    Device removal schema
+    """
+
+    model_config = ConfigDict(
+        populate_by_name=True,
+    )
+    device_name: str
+    """
+    The name of the device.
```

Comment:
please, add also other fields, consumers can be interested in them and they can be optional - it is cheap for us